.github/workflows/simulation_test.yaml

```diff
@@ -0,0 +1,193 @@
+        cmake --build . --parallel
+        cmake --build . --target install
+        echo "SONATAREPORT_DIR=$(pwd)/install" >> $GITHUB_ENV
+
+    - name: Cache NEURON
+      id: cache-neuron
+      uses: actions/cache@v3
+      env:
+        cache-name: cache-neuron
+      with:
+        path: nrn
+        key: ${{ matrix.os }}-neuron-${{ env.NEURON_BRANCH }}-${{ env.NEURON_COMMIT_ID }}-py${{ matrix.python-version }}
+
+    - name: Install NEURON
+      if: steps.cache-neuron.outputs.cache-hit != 'true'
+      run: |
+        sudo apt-get install flex libfl-dev bison ninja-build
+        if [[ ${{ env.NEURON_BRANCH }} == 'master' ]]; then
+          git clone --branch=${{ env.NEURON_BRANCH }} https://github.com/neuronsimulator/nrn.git
+          cd nrn
+          git checkout ${{ env.NEURON_COMMIT_ID }}
+          cd ..
+        else
+          git clone --branch=${{ env.NEURON_BRANCH }} https://github.com/neuronsimulator/nrn.git --depth=1
+        fi
+        . ./venv/bin/activate
+        python -m pip install --upgrade pip -r nrn/nrn_requirements.txt
+        cmake -B nrn/build -S nrn -G Ninja \
+          -DPYTHON_EXECUTABLE=$(which python) \
+          -DCMAKE_INSTALL_PREFIX=$(pwd)/nrn/build/install \
+          -DNRN_ENABLE_MPI=ON \
+          -DNRN_ENABLE_INTERVIEWS=OFF \
+          -DNRN_ENABLE_CORENEURON=ON \
+          -DCMAKE_C_COMPILER=gcc \
+          -DCMAKE_CXX_COMPILER=g++ \
+          -DCORENRN_ENABLE_REPORTING=ON \
+          -DCMAKE_PREFIX_PATH=${{ env.SONATAREPORT_DIR }}
+        cmake --build nrn/build --parallel
+        cmake --build nrn/build --target install
+
+    - name: Install h5py
+      if: steps.cache-venv.outputs.cache-hit != 'true'
+      run: |
+        . ./venv/bin/activate
+        CC=mpicc CXX=mpic++ pip install --no-binary=h5py h5py
+
+    - name: Install neurodamus
+      run: |
+        . ./venv/bin/activate
+        git clone --branch=${{ env.PY_NEURODAMUS_BRANCH }} https://github.com/BlueBrain/neurodamus.git
+        cd neurodamus
+        pip install .
+
+    - name: Build models
+      run: |
+        export PATH=$(pwd)/nrn/build/install/bin:$PATH
+        # Clone neurodamus-models repository
+        git clone --branch=${{ env.NEURODAMUS_MODELS_BRANCH }} https://github.com/BlueBrain/neurodamus-models.git
+        mkdir -p neurodamus/tests/share/mod
+        cp -L -r neurodamus-models/neocortex/mod/v5/* neurodamus/tests/share/mod
+        cp -L -r neurodamus-models/neocortex/mod/v6/* neurodamus/tests/share/mod
+        cp neurodamus/neurodamus/data/mod/*.mod neurodamus/tests/share/mod/
+        cd neurodamus
+        ./docker/build_neurodamus.sh tests/share/mod
+
+    - name: Example run
+      run: |
+        . ./venv/bin/activate
+        if [ ${{ matrix.python-version }} == 3.9 ]; then
+          export RDMAV_FORK_SAFE=1
+        fi;
+        export PYTHONPATH=$(pwd)/nrn/build/lib/python:$PYTHONPATH
+        cp neurodamus/neurodamus/data/hoc/* neurodamus/tests/share/hoc/
```

Comment:
The `*Helper.hoc` files in the `test/share/hoc` are copied long time ago, I need to check if they are still needed. For this test, can you copy the `Helper.hoc` directly from `neurodamus-models`?

Reply:
do you mean to copy the neurodamus-models/neocortex/hoc/* neurodamus/tests/share/hoc/ to overwrite them before copying the data/hoc ones?
```
cp  neurodamus-models/neocortex/hoc/* neurodamus/tests/share/hoc/
cp neurodamus/neurodamus/data/hoc/* neurodamus/tests/share/hoc/
```

Reply:
yes!